### PR TITLE
fixes for node 0.5.x/0.6.x - #226

### PR DESCRIPTION
**Pull request by brentlintner**
- remove use of sys module (deprecated)
- copy lib/\* to node_modules before running tests in node
- fixed a constantly failing unit test

### CODE REVIEW

File: test/unit/w3c/geolocation.js [hunk 1]
```diff
@@ -94,12 +94,16 @@ describe("w3c_geolocation", function () {
         });
 
         it("calls the success callback on the given interval", function () {
-            var watch = geolocation.watchPosition(success, error, {frequency: 10});
-            waits(39);
-            runs(function () {
-                expect(success.callCount).toBe(3);
-                geolocation.clearWatch(watch);
+            spyOn(window, "setInterval").andCallFake(function (callback, frequency) {
```
Comment (ndaversa) [inline test/unit/w3c/geolocation.js:97]: Oh WIN!
Nice.

**Comment by ndaversa** [comment]
@brentlintner , 

Just looked this over, seems good to me.

You okay if I merge this?

**Comment by brentlintner** [comment]
Hey man that would be sweet. The only thing I forgot to do I just realized is check the bash commands on Mac, but I'm pretty sure cp's -rf flags are common though. Thanks!

**Comment by brentlintner** [comment]
... on second thought I'm going to double check. one sec.

**Comment by brentlintner** [comment]
Checked out on my Macbook (Lion), and it force copies properly. seems like -R and -r go hand in hand.

**Pull request merged**